packages/server/graphql/mutations/helpers/generateGroupSummaries.ts

```diff
@@ -0,0 +1,34 @@
+import extractTextFromDraftString from 'parabol-client/utils/draftjs/extractTextFromDraftString'
+import getRethink from '../../../database/rethinkDriver'
+import OpenAIServerManager from '../../../utils/OpenAIServerManager'
+import {DataLoaderWorker} from '../../graphql'
+
+const generateGroupSummaries = async (
+  meetingId: string,
+  dataLoader: DataLoaderWorker,
```

Comment:
nit: maybe put dataloader param last? (Could be my personal feeling)

Reply:
I also like the `dataLoader` to be last! Here I've put the `facilitatorUserId` last only because we use it to check the feature flag. Once the feature is shipped to everyone, we'll be able to remove that param so `dataLoader` will be last

Reply:
Make sense